### PR TITLE
blog:  The Security Implications Of Passwordless Authentication

### PR DESCRIPTION
**Pull request by sixhobbits**

### CODE REVIEW

**Review by bradmccarty — COMMENTED** [review]
The content is killer. Absolutely above and beyond what I'd hoped to see. One request: change the URL to passwordless-authentication-security.mdx please. After that I'm happy to get it published.

**Review by bradmccarty — APPROVED** [review]

File: astro/src/content/blog/passwordless-authentication.mdx [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+
+However, this process does raise a few questions about hardware restrictions and the capacity to store keys. As discussed earlier, current models of security keys like Yubico’s have a limitation on the number of resident keys they can store. Once maxed out, users would need a second security key for additional accounts. It's a limitation crucial to keep in mind when considering deploying passwordless authentication, but it shouldn't impact those who use it for personal accounts or even professionals handling a limited number of prioritized accounts. As technology continues to advance rapidly, it’s likely these storage limitations will expand over time too.
+
+Unfortunately, the decision of whether to use resident versus non-resident keys isn't up to the user. Web services that offer passwordless authentication will determine the key type used based on whether they require a username to log in.
+
+Mobile phones have ample space for resident keys, so for most users, this limitation would not apply.
+
+## How passwordless authentication Protects Against Common Cybersecurity Threats
```
Comment (mooreds) [inline astro/src/content/blog/passwordless-authentication.mdx:84]:
```suggestion
## How Passwordless Authentication Protects Against Common Cybersecurity Threats
```

File: astro/src/content/blog/passwordless-authentication.mdx [hunk 2]
```diff
@@ -0,0 +1,160 @@
+---
+publish_date: 2023-11-10
+title: The Security Implications Of Passwordless Authentication
+description: Learn about passwordless authentication
+authors: Dan Moore
```
Comment (mooreds) [inline astro/src/content/blog/passwordless-authentication.mdx:5]: Don't care who this is, happy to have it ghostwritten. Also happy to have it be one of Ritza authors.

**Review by mooreds — COMMENTED** [review]
Two small suggested changes, other than that looks good.

**Comment by sixhobbits** [comment]
@mooreds @bradmccarty - renamed the file and changed the author.

Note that the image is re-used from a previous post -- let me know if you need anything from us re that.

**Review by bradmccarty — APPROVED** [review]

**Pull request merged**